carbon-design-system/ibm-security
# docs(addon-docs): prevent MDX scrolling while navigating stories

## Proposed change

Add workaround to prevent scrolling in MDX stories as per https://github.com/storybookjs/storybook/issues/10983

## Testing instructions

Verify scroll position navigates to the top when navigating between layout modules documentation - https://modular-layouts-documentation--ibm-security.netlify.app/?path=/docs/patterns-unstable-layout-modules-actionbarmodule--default
